doc/sphinx/Pacemaker_Development/faq.rst

```diff
@@ -73,22 +72,37 @@ Frequently Asked Questions
    single: git; commit message
 
 :Q: How should I format my Git commit messages?
-:A: See existing examples in the git log. The first line should look like
-    +change-type: affected-code: explanation+ where +change-type+ should be
-    +Fix+ or +Bug+ for most bug fixes, +Feature+ for new features, +Log+ for
-    changes to log messages or handling, +Doc+ for changes to documentation or
-    comments, or +Test+ for changes in CTS and regression tests. You will
-    sometimes see +Low+, +Med+ (or +Mid+) and +High+ used instead for bug fixes,
-    to indicate the severity. The important thing is that only commits with
-    +Feature+, +Fix+, +Bug+, or +High+ will automatically be included in the
-    change log for the next release. The +affected-code+ is the name of the
-    component(s) being changed, for example, +controller+ or
-    +libcrmcommon+ (it's more free-form, so don't sweat getting it exact). The
-    +explanation+ briefly describes the change. The git project recommends the
-    entire summary line stay under 50 characters, but more is fine if needed for
-    clarity. Except for the most simple and obvious of changes, the summary should
-    be followed by a blank line and then a longer explanation of 'why' the change
-    was made.
+:A: An example is "Feature: scheduler: wobble the frizzle better".
+   
+    * The first part is the type of change, used to automatically generate the
+      change log for the next release. Commit messages with the following will
+      be included in the change log:
+
+      * **Feature** for new features
+      * **Fix** for bug fixes (**Bug** or **High** also work)
+      * **API** for changes to the public API
+
+      Everything else will *not* automatically be in the change log, and so
+      don't really matter, but types commonly used include:
+
+      * **Log** for changes to log messages or handling
+      * **Doc** for changes to documentation or comments
+      * **Test** for changes in CTS and regression tests
+      * **Low**, **Med**, or **Mid** for bug fixes not significant enough for a
+        change log entry
+      * **Build** for build process changes
```

Comment:
It might also be worth mentioning Refactor here.  That seems to be used pretty often.  At least, I'm using it a lot.

Reply:
Ah yes